form-val.js

```diff
@@ -10,3 +10,30 @@ function validate(e) {
     err.innerHTML = 'Email cannot be uppercase';
   }
 }
+
+const fullNameForm = document.getElementById('name');
+const emailForm = document.getElementById('email');
+const messageForm = document.getElementById('message');
+
+function handleChange() {
+  const formData = {
+    fullName: fullNameForm.value,
+    email: emailForm.value,
+    comment: messageForm.value,
+  };
+  localStorage.setItem('form', JSON.stringify(formData));
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const getFormValue = localStorage.getItem('form');
+  if (getFormValue) {
+    const formObject = JSON.parse(getFormValue);
+    fullNameForm.value = formObject.fullName;
+    emailForm.value = formObject.email;
+    messageForm.value = formObject.comment;
+  }
+});
```

Comment:
- [ ] As mentioned by previous reviewer, your form should save the date for user after the page refresh and it would be something like the one in this [loom video](https://www.loom.com/share/0384a97910564945a5becaeae3d403b8)